upgrade/breaking-changes/4-0-0 .md

```diff
@@ -0,0 +1,308 @@
+page_title: Breaking Changes in 4.0.0
+slug: changes-in-4-0-0
+position: 1
+---
+
+# Breaking Changes in 4.0.0
+
+## Common Changes
+
+### Supported .NET versions
+
+Microsoft drops the support of `.NET Core 3.1.x` on December 13, 2022. From the 4.0.0 release, Telerik UI for Blazor will support `.NET 6` and newer versions.
+
+### Stretch enum value
+
+* [Card Actions `Layout`]({%slug card-actions%}#layout)
+* [Dialog Action `Buttons Layout`]({%slug dialog-action-buttons%})
+* [Grid PopUp `Form Buttons Layout`]({%slug components/grid/editing/popup%}#edit-form-customization)
+* [Scheduler PopUp `Form Buttons Layout`]({%slug scheduler-edit-popup-customization%}#edit-form-customization)
+* [TreeList PopUp `Form Buttons Layout`]({%slug treelist-editing-popup%}#edit-form-customization)
+
+>caption Stretched enum value in UI for Blazor up to version 3.8.0 and after version 4.0.0
+
+<table>
+    <thead><tr>
+        <th>UI for Blazor 3.8.0</th>
+        <th>UI for Blazor 4.0.0</th>
+    </tr></thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;CardActions Layout="@CardActionsLayout.Stretched /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;CardActions Layout="@CardActionsLayout.Stretch" /&gt;
+</code></pre>
+            </td>
+        </tr>  
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikDialog ButtonsLayout="@DialogButtonsLayout.Stretched" /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikDialog ButtonsLayout="@DialogButtonsLayout.Stretch" /&gt;
+</code></pre>
+            </td>
+        </tr>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;GridPopupEditFormSettings ButtonsLayout="FormButtonsLayout.Stretched" /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;GridPopupEditFormSettings ButtonsLayout="FormButtonsLayout.Stretch" /&gt;
+</code></pre>
+            </td>
+        </tr>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;SchedulerPopupEditFormSettings ButtonsLayout="FormButtonsLayout.Stretched" /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;SchedulerPopupEditFormSettings ButtonsLayout="FormButtonsLayout.Stretch" /&gt;
+</code></pre>
+            </td>
+        </tr>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TreeListPopupEditFormSettings ButtonsLayout="FormButtonsLayout.Stretched" /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TreeListPopupEditFormSettings ButtonsLayout="FormButtonsLayout.Stretch" /&gt;
+</code></pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Async Method Suffix
+
+Added the `Async` suffix to the signature of asynchronous methods
```

Comment:
"signature" is correct, but why not say it simpler? Do we expect other methods here, or should we focus on `SetState` only?

```suggestion
Added an `Async` suffix to the name of asynchronous methods.
```